### DISCUSSION TITLE
Previewing Presentation in jupyter-notebook failing

### DISCUSSION THREAD

**Discussion by kurianbenoy** [quarto-dev/quarto-cli#5591] [Q&A] [answered]
Hi team, 

I wanted to reach out regarding an issue I encountered while attempting to follow the Jupyter notebooks presented in the nbdev tutorial at JupyterCon 2023.

Specifically, I have been working with the Jupyter notebook located at the following GitHub repository: https://github.com/fastai/jupytercon-2023/tree/main/_quarto_demo.

Upon running the command quarto preview presentation.ipynb on my local machine, I received the following error message:

```
Error running filter shinylive:
Could not find executable shinylive
```

If you are interested to see the full output, check this [gist document](https://gist.github.com/kurianbenoy/7963b99cea98170d2b6e4043fe9b7a04). I would really appreciate any help in fixing this issue for me in the local.

I guess I might be missing some necessary dependencies. Also please point me to some relevant documentations, where I can learn more.

**Comment by mcanouil** [comment, marked as answer]
You need the shinylive Quarto extension and dependencies, see <https://github.com/quarto-ext/shinylive>.

**Reply by kurianbenoy** [reply]
Thank you
